File: Clase 16/JAVA/ejemploAnimales/Gato.java
package ejemploAnimales;

public class Gato extends Animal{
    protected String tipoPelaje;

    public Gato(int id, String nombre, String especie, String tipoPelaje){
        super(id, nombre, especie);
        this.tipoPelaje = tipoPelaje;
    }

    public void haceMiau(){
        System.out.println(this.nombre + " hace MIUAUUU!!!!");
    }
}

File: Clase 16/PYTHON/ejemplo_animales/Gato.py
from Animal import Animal

class Gato(Animal):

    def __init__(self, id, nombre, especie, tipo_pelaje) -> None:
        super().__init__(id, nombre, especie)
        self.tipo_pelaje = tipo_pelaje
    
    def hace_miau(self):
        print(f"{self.nombre} hace MIAUUU!!!!")